flink-connectors/flink-connector-rabbitmq/README.md

```diff
@@ -9,3 +9,7 @@ nor packages binaries from the "RabbitMQ AMQP Java Client".
 Users that create and publish derivative work based on Flink's
 RabbitMQ connector (thereby re-distributing the "RabbitMQ AMQP Java Client")
 must be aware that this may be subject to conditions declared in the Mozilla Public License 1.1 ("MPL"), the GNU General Public License version 2 ("GPL") and the Apache License version 2 ("ASL").
+
+# This version provides a mechanism to handle AMQ Messaging features
```

Comment:
I think the Javadocs of the constructors are in better shape  than the contents of the `README.md`. I think we can drop this paragraph for now. What do you think?

Reply:
ping